cypress/support/pages/tasks/pageTasks.js

```diff
@@ -7,31 +7,45 @@ class Tasks {
   static #taskOverviewStudent = '[class="task-dashboard-student"]'
   static #groupSubmissionCheckbox = '[id="teamSubmissions"]'
   static #draftCheckbox = '[data-testid="private-checkbox"]'
-  static #visibilityStartDateInput = '[data-testid="form-datetime-input-availableDate"]'
+  static #visibilityStartDateInput =
+    '[data-testid="form-datetime-input-availableDate"]'
   static #visibilityDueDateInput = '[data-testid="form-datetime-input-dueDate"]'
   static #publicSubmissionsCheckbox = '[id="publicSubmissionsCheckbox"]'
-  static #dialogConfirmButton = '[data-testid="task-publicSubmissions-dialog-confirm"]'
-  static #dialogCancelButton = '[data-testid="task-publicSubmissions-dialog-cancel"]'
-  static #dialogCancelDeletionTaskButtons = '#modal-delete-homework-footer > .btn-close'
-  static #dialogConfirmDeletionTaskButtons = '#modal-delete-homework-footer > .btn-submit'
+  static #dialogConfirmButton =
+    '[data-testid="task-publicSubmissions-dialog-confirm"]'
+  static #dialogCancelButton =
+    '[data-testid="task-publicSubmissions-dialog-cancel"]'
+  static #dialogCancelDeletionTaskButtons =
+    '#modal-delete-homework-footer > .btn-close'
+  static #dialogConfirmDeletionTaskButtons =
+    '#modal-delete-homework-footer > .btn-submit'
   static #taskDetailsTab = '[id="extended"]'
   static #submissionTab = '[id="submission-tab-link"]'
   static #submissionsTab = '[id="submissions-tab-link"]'
   static #taskDetailsEditButton = '[data-testid="task-details-btn-edit"]'
-  static #fileUploadButtonDisabled = '[data-testid="tasks-edit-fileupload"]'
-  static #fileUploadButtonEnabled = '[data-testid="tasks-edit-fileupload"]'
-  static #fileUploadInput = '[data-testid="tasks-edit-fileupload-input"]'
-  static #filesSection = '[data-testid="tasks-edit-section-files"]'
+  static #fileUploadButtonDisabled = '[data-testid="fileupload-button"]'
```

Comment:
Here I see that both selector have same value. Maybe use some name which corresponds both ways.

``` 
static #fileUploadButtonDisabled = '[data-testid="fileupload-button"]'
static #fileUploadButtonEnabled = '[data-testid="fileupload-button"]'
```

Reply:
Yes, I saw that, too. There were different ids, but after changes in the UI and adaptions of Ankush they are now the same.
The problem is, that this check, if the button is en- or disabled has to be adapted, that's why I left it as it is.
I'll create a ticket for correction of this test step.

Reply:
Then its fine from myside